### ISSUE TITLE
xxl breakpoint size?

### ISSUE THREAD

**Issue by ixahmedxi** [halfmoonui/halfmoon#19]
Hello,

First of all awesome framework, great job! I have a small suggestion, with the increased resolution of screen, it seems like 1200px as a max breakpoint is a bit too small. Bootstrap v5 introduces a new breakpoint (xxl) which is 1400px. I think this breakpoint could be a great addition as that it will be nicer on high resolution screens. I can help out with creating this.

Thanks!

**Comment by halfmoonui** [comment]
Hey, thanks for bringing this up. Halfmoon auto-scales up everything when the screen size gets bigger than `1600px`. You can see it in this image: [Image link](https://res.cloudinary.com/halfmoon-ui/image/upload/v1597446657/Screen_Shot_2020-08-15_at_5.09.51_AM_tu8foy.png)

Do you feel this removes the need for the `xxl` breakpoint you are suggesting? I would love some more opinions too if possible.

**Comment by halfmoonui** [comment]
Only adding an extra extra large `.container` does not make sense to me. It has to be a full breakpoint if it is to be added.

**Comment by ixahmedxi** [comment]
yes of course not only for `.container` but for anything that gets a restricted widths based on breakpoints like `.container`.
